RealmSwift/ThreadSafeReference.swift

```diff
@@ -122,6 +122,73 @@ public protocol ThreadConfined {
     }
 }
 
+/**
```

Comment:
This should have a MARK above it:
```
// MARK: ThreadSafe propertyWrapper
```

Reply:
Should be changed in latest commit. Not sure if github ui picked it up as outdated.